tpchgen/src/generators.rs

```diff
@@ -131,7 +131,7 @@ impl Iterator for NationGeneratorIterator {
             // n_regionkey
             n_regionkey: self.nations.get_weight(self.index) as i64,
             // n_comment
-            n_comment: self.comment_random.next_value(),
+            n_comment: self.comment_random.next_value().to_string(),
```

Comment:
As mentioned in the description, there is still a lot of `String` copying going on, but after this PR is it more explicit.

Follow on PRs will remove these calls to `to_string` and actually make the code faster